accounts-db/src/ancient_append_vecs.rs

```diff
@@ -441,13 +441,14 @@ impl AccountsDb {
             StoreReclaims::Ignore,
         ));
 
-        write_ancient_accounts.metrics.accumulate(&ShrinkStatsSub {
+        let shrink_stats = ShrinkStatsSub {
             store_accounts_timing,
-            rewrite_elapsed_us,
-            create_and_insert_store_elapsed_us,
-            unpackable_slots_count: 0,
-            newest_alive_packed_count: 0,
-        });
+            rewrite_elapsed_us: Saturating(rewrite_elapsed_us),
+            create_and_insert_store_elapsed_us: Saturating(create_and_insert_store_elapsed_us),
+            ..ShrinkStatsSub::default()
+        };
+        write_ancient_accounts.metrics.accumulate(&shrink_stats);
```

Comment:
*minor*

I think it is fine to use a temporary value, as the stats are not used for anything else.
And the indentation level is identical in both cases.

Reply:
Fair enough, not positive why I changed it 😅; done in https://github.com/anza-xyz/agave/pull/523/commits/75b44f2f9ff1aac4da079a5ab96ce5846e7cb202